src/luarocks/command_line.lua

```diff
@@ -10,6 +10,17 @@ local path = require("luarocks.path")
 local dir = require("luarocks.dir")
 local deps = require("luarocks.deps")
 local fs = require("luarocks.fs")
+--- This is to install platform-specific fs functions.
+-- Since there were cyclic dependencies between fs.* modules
+-- and it was very hard to compile luarocks into Tarantool
+-- with them (an artifact of current preloader implementation),
+-- they were broken down. As a consequence, we have to manually
+-- load the platform-specific fs functions here, or they
+-- just won't work.
+require("luarocks.fs.unix")
```

Comment:
Now we don't pcall it, so unix is mandatory. Tarantool doesn't support other OSes, so it can be considered ok. But I don't like the idea mix non-so-related changes as drop win32 support and prevent cycling in require. It is not a blocker, I just think that we can prevent cyclic deps w/o these changes and Occam's razor push me dive into this nitpicking. Feel free to ignore this comment except proper commenting in the commit message any changes of the behaviour.

Reply:
Squashed review fixes and reworded commit message to mention this.